client/src/main/scala/drt/client/components/QueueChartComponent.scala

```diff
@@ -15,11 +15,12 @@ object QueueChartComponent {
   case class Props(queue: Queue,
                    queueSummaries: List[(Long, Map[Queue, CrunchMinute])],
                    sla: Int,
+                   interval: Int,
                    deskType: DeskType)
 
   val component: Component[Props, Unit, Unit, CtorType.Props] = ScalaComponent.builder[Props]("QueueChart")
     .render_P { props =>
-      val labels: Seq[String] = (0 until 96).map(m => SDate("2022-08-17T23:00").addMinutes(m * 15).toHoursAndMinutes)
+      val labels: Seq[String] = (0 until 96).map(m => SDate("2022-08-17T23:00").addMinutes(m * props.interval).toHoursAndMinutes)
```

Comment:
We can't use `0 until 96` when the interval is 60 minutes. This now needs to be calculated according to the interval